ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json

```diff
@@ -1,10 +1,7 @@
 [
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "DiedBefore", "ListElement": "They died before the deceased"},
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "DiedAfter", "ListElement": "They died after the deceased"},
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "PowerReserved", "ListElement": "They're holding power reserved"},
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "Renunciation", "ListElement": "They have renounced"},
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "PowerOfAttorney", "ListElement": "They have appointed or wish to appoint another person to act as their attorney"},
-  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "MentallyIncapable", "ListElement": "They lack capacity to act as executor"},
+  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "PowerReserved", "ListElement": "They are holding power reserved", "DisplayOrder": 1},
+  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "Renunciation", "ListElement": "They have renounced", "DisplayOrder": 2},
+  {"LiveFrom": "01/01/2017", "ID": "notApplyingExecutorReasonFixedList", "ListElementCode": "PowerOfAttorney", "ListElement": "They have appointed or wish to appoint another person to act as their attorney", "DisplayOrder": 3},
```

Comment:
I remember something about these fixed lists and there was uncertainty about what should be involved in it

Reply:
Oh I forgot! Thanks Kleo!